magento1-module/app/code/local/Divante/VueStorefrontBridge/Model/Api/Request.php

```diff
@@ -127,6 +127,10 @@ public function currentQuote($request)
                 $tokenData = JWT::decode($cartId, $secretKey, 'HS256');
                 $this->quote =  Mage::getModel('sales/quote')->load($tokenData->cartId);
             }
+
+            if (!(bool)$this->quote->getId()) {
+                $this->quote = null;
```

Comment:
The case you try to check for will probably yield :`PHP Fatal error: Uncaught Error: Call to a member function getId() on boolean.` as in that case `$this->quote` would be false i think?

Reply:
I would check if the ID is NULL as when don't need the cast to bool. 

And the error you mean does not happen I have tried it with like an empty sting, a number that I not have as a quote or like null as argument.